site/docs/features/intro.md

```diff
@@ -0,0 +1,107 @@
+# Introduction
+
+Nessie is an OSS service and libraries that enable you to maintain multiple versions 
+of your data and leverage Git-like Branches & Tags for your Data Lake. Nessie enhances the following 
+table formats with version control techniques:
+
+* Apache Iceberg Tables ([more](../tables/iceberg.md))
+* Delta Lake Tables ([more](../tables/deltalake.md))
+* Hive Metastore Tables ([more](../tables/hive.md))
+* SQL Views ([more](../tables/views.md))
+
+## Basic Concepts
+
+Nessie is heavily inspired by Git. The main concepts Nessie exposes map directly to 
```

Comment:
For consistency across both this and the other files I'd go with "Git" rather than "git".

Reply:
Applied